rmc/utils/mpc.py

```diff
@@ -305,9 +309,16 @@ def prepare_pop_path_ht(
     gnomad_ht = gnomad_ht.filter(gnomad_ht.freq[0].AF > af_threshold)
     gnomad_ht = gnomad_ht.annotate(pop_v_path=1)
 
-    logger.info("Joining ClinVar and gnomAD HTs...")
+    logger.info("Joining ClinVar and gnomAD HTs and filtering out overlaps...")
     ht = clinvar_ht.select("pop_v_path").union(gnomad_ht.select("pop_v_path"))
-    ht = ht.checkpoint(f"{TEMP_PATH}/joint_clinvar_gnomad.ht", overwrite=True)
+    # Remove variants that are in both the benign and pathogenic set
+    counts = ht.group_by(*ht.key).aggregate(n=hl.agg.count())
+    ht = ht.anti_join(counts.filter(counts.n > 1))
```

Comment:
```suggestion
    ht = ht.anti_join(overlap)
```
(if you accept the suggestion above)